build.gradle

```diff
@@ -12,7 +12,7 @@ plugins {
 	id 'checkstyle'
 	id 'jacoco'
 	id 'pmd'
-	id 'com.github.ben-manes.versions' version '0.25.0'
+	id 'com.github.ben-manes.versions' version '0.20.0'
```

Comment:
later config from master which was implemented after this branch was created from demo needs to be retained